x-pack/plugins/cloud_security_posture/public/pages/rules/rules_flyout.tsx

```diff
@@ -4,23 +4,165 @@
+    }),
+    description: '',
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.sectionsLabel', {
+      defaultMessage: 'Sections',
+    }),
+    description: '',
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.profileApplicabilityLabel', {
+      defaultMessage: 'Profile Applicability',
+    }),
+    description: '',
+  },
+
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.profileApplicabilityLabel', {
+      defaultMessage: 'Profile Applicability',
+    }),
+    description: rule.attributes.description,
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.auditLabel', {
+      defaultMessage: 'Audit',
+    }),
+    description: '',
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.referencesLabel', {
+      defaultMessage: 'References',
+    }),
+    description: '',
+  },
+];
+
+const getRemediationCard = (rule: RuleSavedObject): EuiDescriptionListProps['listItems'] => [
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.remediationLabel', {
+      defaultMessage: 'Remediation',
+    }),
+    description: rule.attributes.remediation,
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.impactLabel', {
+      defaultMessage: 'Impact',
+    }),
+    description: rule.attributes.impact,
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.defaultValueLabel', {
+      defaultMessage: 'Default Value',
+    }),
+    description: rule.attributes.default_value,
+  },
+  {
+    title: i18n.translate('xpack.csp.rules.ruleFlyout.rationaleLabel', {
+      defaultMessage: 'Rationale',
+    }),
+    description: rule.attributes.rationale,
+  },
+];
+
+export const RuleFlyout = ({ onClose, rule, toggleRule }: RuleFlyoutProps) => {
+  const [tab, setTab] = useState<RuleTab>('overview');
+
   return (
     <EuiFlyout ownFocus={false} onClose={onClose} outsideClickCloses>
       <EuiFlyoutHeader>
-        <h1>{rule.attributes.name}</h1>
+        <EuiTitle size="l">
+          <EuiTextColor color="primary">
+            <h2>{rule.attributes.name}</h2>
+          </EuiTextColor>
+        </EuiTitle>
         <EuiSpacer />
+        <EuiTabs>
+          {tabs.map((tabName) => (
+            <EuiTab
+              key={tabName}
+              isSelected={tab === tabName}
+              onClick={() => setTab(tabName)}
+              style={{ textTransform: 'capitalize' }}
+              disabled={tabName === 'rego code'}
+            >
+              {tabName}
+            </EuiTab>
+          ))}
+        </EuiTabs>
       </EuiFlyoutHeader>
-      <EuiFlyoutBody />
+      <EuiFlyoutBody>
+        {tab === 'overview' && <RuleOverviewTab rule={rule} toggleRule={() => toggleRule(rule)} />}
+        {tab === 'remediation' && (
+          <EuiDescriptionList
+            compressed={false}
+            type="column" // TODO: fix?
+            listItems={getRemediationCard(rule)}
+            style={{ flexDirection: 'column' }}
```

Comment:
Why is this needed?

Reply:
because `type=column` doesn't add the flex direction

Reply:
I removed both `type` and `style` prop and it looks fine. Am I missing something?
<img width="1840" alt="Screen Shot 2022-03-21 at 14 11 07" src="https://user-images.githubusercontent.com/94899776/159258446-ee40aff9-166c-4d8a-a98b-496278dd53cf.png">

Reply:
seems to work perfect. thanks!